beta/src/pages/learn/your-first-component.md

```diff
@@ -1,24 +1,24 @@
 ---
-title: Your First Component
+title: आपका पहला कौम्पोनॅन्ट
 ---
 
 <Intro>
 
-Components are one of the core concepts of React. They are the foundation upon which you build user interfaces (UI), which makes them the perfect place to start your React journey!
+कौम्पोनॅन्ट React की मुख्य अवधारणाओं में से एक हैं। वे नींव हैं जिस पर आप यूजर इंटरफेस (UI) का निर्माण करते हैं, जो उन्हें आपकी React यात्रा शुरू करने के लिए एक आदर्श स्थान बनाता है!
 
 </Intro>
 
 <YouWillLearn>
 
-* What a component is
-* What role components play in a React application
-* How to write your first React component
+- एक कौम्पोनॅन्ट क्या है
+- React एप्लिकेशन में कौम्पोनॅन्ट क्या भूमिका निभाते हैं
+- अपना पहला React कौम्पोनॅन्ट कैसे लिखें
 
 </YouWillLearn>
 
-## Components: UI building blocks {/*components-ui-building-blocks*/}
+## कौम्पोनॅन्ट: UI बिल्डिंग ब्लॉक्स {/* components-ui-building-blocks */}
 
-On the Web, HTML lets us create rich structured documents with its built-in set of tags like `<h1>` and `<li>`:
+Web पर, HTML हमें `<h1>` और `<li>` जैसे टैग के अंतर्निहित सेट के साथ समृद्ध संरचित दस्तावेज़ बनाने देता है:
```

Comment:
`समृद्ध संरचित दस्तावेज़` => `रिच स्ट्रक्चर्ड डाक्यूमेंट्स`

hindi translation seems a bit complex. lets keep it in hinglish